tsconfig.json

```diff
@@ -4,7 +4,8 @@
         "types": [
             "node"
         ],
-        "target": "es2016"
+        "target": "es2016",
+        "experimentalDecorators": true

```

Comment:
```@suite``` and ```@test``` are ,markers mocha relies on to discover a test stub but typescript does not recognize. However, tsc has some experimental support for such markers, hence the flag to appease it. @olydis do weigh in with your thoughts here.